### ISSUE TITLE
Scratch layer only restoring last window

### ISSUE THREAD

**Issue by bjeanes** [paperwm/PaperWM#133]
I upgraded to get fix for #132 and since doing so Super+ESC will hide all scratch windows but will only restore what appears to be the most recently active one. I usually keep Spotify, Slack, and other comms apps in scratch layer, but my usual flow doesn't work if I have to go back to Alt+Tab to find the right window. Is this a regression or did the behaviour of scratch layer change (e.g. perhaps that is now the shortcut to specifically get latest window and there is a new shortcut to get full layer?)

**Comment by hedning** [comment]
Yeah, we changed it a while back, `super-shift-escape` will now activate the whole layer (can be rebound in the preference gui). Ideally we should make `super-escape` remember which windows was last used though.

**Comment by bjeanes** [comment]
Hmm I am sure this has already been discussed and tabled, but from a UX perspective, it's a bit odd that super-escape hides _all_ windows then. The super+shift+escape is pretty awkward combo. In any case, it's remappable so not a big deal...

**Comment by bjeanes** [comment]
Thanks!

**Comment by hedning** [comment]
Yeah, it's a bit weird and we went a bit back and forth without finding a great solution. IIRC we landed on symmetry with unfloat window ( `super-ctrl-escape`) which really didn't work that well if we didn't minimize the other scratch windows at the same time.

But yeah, I think making super-escape toggle between the last used/open scratch windows makes sense, that way it's possible to prune away windows that aren't used much with minimize. I'll try to get that done pretty soon, should be straightforward.
